docs/conf/portland/2020/opportunity-grants.rst

```diff
@@ -0,0 +1,57 @@
+:template: {{year}}/generic.html
+
+Opportunity grants
+==================
+
+The grant program for WTD Portland 2020 supports people who would otherwise not be able to attend the conference by providing free tickets and covering travel and accommodation costs. Applications are open to anyone who wants to attend Write the Docs.
+
+To apply, fill in the form below. Note that not all applications will be accepted due to limits on funds. Please do not book any non-refundable travel until you have a confirmation that your application is accepted.
+
+We prioritize applications based on the overall impact that granting an application will have on the applicant, the Write the Docs community, and the applicant's wider community and country, compared to others.
+
+Grant applicants, like all other participants in the Write the Documents community, are required to conform to the Code of Conduct: https://www.writethedocs.org/code-of-conduct/.
+
+.. contents::
+    :local:
+    :depth: 1
+    :backlinks: none
+
+What is covered
+----------------
+
+All grants already include a free conference ticket, social event, and breakfast and lunch on conference days.
```

Comment:
I don't think we always want to include a free ticket. What is people just need a discounted ticket?